<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="TreeBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Telem-Tree</string>
        </property>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>231</width>
           <height>571</height>
          </rect>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Instrument</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Demodulation</string>
          </property>
          <item>
           <property name="text">
            <string>Summary</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Raw Hex Frames</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Parity Check</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Spacecraft</string>
          </property>
          <item>
           <property name="text">
            <string>Spacecraft ID</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Timestamps</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>HIRS</string>
          </property>
          <item>
           <property name="text">
            <string>Channels</string>
           </property>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Telemetry</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>DCS</string>
          </property>
          <item>
           <property name="text">
            <string>DCS Summary</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>CPU</string>
          </property>
          <item>
           <property name="text">
            <string>CPU A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CPU B</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>SEM</string>
          </property>
          <item>
           <property name="text">
            <string>MEPED</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TED</string>
           </property>
          </item>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>The Good Stuff</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="Demod">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Demodulation Details&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;The modulation scheme is a non orthagonal BPSK, also known as a preserved carrier PSK.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;The phase shift is between -67 and +67 degrees, which allows the carrier frequency to be preserved and tracked using a simple PLL. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;The bits are manchester coded for ease of clock recovery, which means there are two symbols per bit. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Summary: &lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Dsplays the results of the demodulation&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Raw Hex: &lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Displays the raw minor frames in hex format for visualization or copy/paste into another application&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Parity: &lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Calculates the errors in the bitstream in 5 chunks and verifies that there are at least an even or odd number of errors in each chunk.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="textFormat">
                <enum>Qt::RichText</enum>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Summary">
            <widget class="QTableWidget" name="tableWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>741</width>
               <height>561</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>10</y>
               <width>71</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Summary</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="RawHex">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>RawHex</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPlainTextEdit" name="minorFrameBrowser">
               <property name="lineWrapMode">
                <enum>QPlainTextEdit::NoWrap</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Parity">
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>55</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Parity</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="SpaceCrft">
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>10</y>
               <width>71</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Spacecraft</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="SPC_ID">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>SPC_ID</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QwtPlot" name="SPIDPlot"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="TStamp"/>
           <widget class="QWidget" name="HIRS"/>
           <widget class="QWidget" name="HChans"/>
           <widget class="QWidget" name="HChanAll"/>
           <widget class="QWidget" name="HTlm"/>
           <widget class="QWidget" name="DCS">
            <widget class="QTextEdit" name="textEdit">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>751</width>
               <height>561</height>
              </rect>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;DCS transmitters are ground based transmitters that contain a short payload and geolocation information&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="DSum">
            <widget class="QTextBrowser" name="textBrowser">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>751</width>
               <height>561</height>
              </rect>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Transmitter Summary (lots and lots of stuff and stuff)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="CPU">
            <widget class="QTextBrowser" name="textBrowser_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>751</width>
               <height>561</height>
              </rect>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;CPU Telemetry is still being reverse engineered and is probably impossible to completely understand given the amount of possible magic data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="CPUA"/>
           <widget class="QWidget" name="CPUB"/>
           <widget class="QWidget" name="SEM"/>
           <widget class="QWidget" name="MEPED"/>
           <widget class="QWidget" name="TED"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
